### PR TITLE
Add travis ci

### PR DESCRIPTION
**Pull request by janfreyberg**
This at the moment doesn't test much, but at least tests whether the package can be installed (to catch problems like this one: https://github.com/nipy/niwidgets/pull/41)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +37,8 @@ So far, the widgets support plotting of nifti files, either in `nii` or `nii.gz`
 format. You initialise a widget class like this:
 
 ```
-from niwidgets import NiftiWidget
-my_widget = NiftiWidget('./path/to/file.nii')
+from niwidgets import NiftiWidget my_widget =
```
Comment (satra) [inline README.md]: this should be reverted.

**Comment by arokem** [comment]
Did you mean to remove the requirements file? If so, why?

**Comment by satra** [comment]
@arokem - i think it's because @janfreyberg included the dependencies in setup.py. so theoretically requirements.txt is not needed. pip and python will use the requirements from setup.py if using recent setuptools.

**Comment by janfreyberg** [comment]
Yep - setup.py installs the dependencies so I think the requirements.txt file was superfluous. Also, on a practical note, Travis installs from requirements.txt by default, which meant it didn't catch the missing dependencies

File: README.md (hunk 1 above)
Comment (janfreyberg) [reply README.md]: Nice catch, thanks

File: setup.py [hunk 2]
```diff
@@ -32,12 +33,19 @@
     author_email='jan.freyberg@gmail.com',
     packages=['niwidgets'],
     keywords=['widgets', 'neuroimaging'],
-    install_requires=['ipywidgets', 'nilearn', 'nibabel', 'ipyvolume'],
+    install_requires=['ipywidgets', 'nilearn', 'nibabel', 'ipyvolume',
```
Comment (arokem) [inline setup.py]: Is nilearn a real dependency? I don't see it imported in any of the code. Only in examples.

**Comment by arokem** [comment]
Looks like merging is blocked because your commits are not signed, @janfreyberg. 

https://help.github.com/articles/about-gpg/

Do we really need to enforce signing commits?

**Comment by janfreyberg** [comment]
No, we don't! I thought this meant having a committer email defined, not signing with gpg...

File: setup.py (hunk 2 above)
Comment (janfreyberg) [reply setup.py]: This is true, yep.

**Pull request merged**